GOVERNANCE.md

```diff
@@ -0,0 +1,38 @@
+# in-toto governance
+This document covers the project's governance and committer process.  The
+project consists of the in-toto
+[specification](https://github.com/in-toto/docs) and
```

Comment:
Should this link to the actual specification file?

Reply:
Not sure. But since the repo currently only contains the specification (in different formats) plus a management script, I think it is fine to just link to the repo.